File: Client/src/app/_models/priority.ts
export interface Priority {
    id: number;
    rowGuiid: string;
    priorityName: string;
    timeType: TimeType;
    time: number;
    listIndex: number;
}

export enum TimeType {
    Minutes = 1,
    Hours = 2,
    Days = 3
}

export interface PriorityAdd {
    priorityName: string;
    timeType: TimeType;
    time: number;
    listIndex: number;
}

export const TimeTypeList = [
    {value: 1, name: "Dakika"},
    {value: 2, name: "Saat"},
    {value: 3, name: "Gün"}
  ];

File: API/Entities/Priority.cs
using API.enums;

namespace API.Entities
{
    public class Priority
    {
        public int Id { get; set; }
        public string RowGuid { get; set; }
        public string PriorityName { get; set; }

        public TimeType TimeType { get; set; }
        public int Time { get; set; }
        public int ListIndex { get; set; }

    }
}